modules/apps/29-fee/types/keys.go

```diff
@@ -37,3 +40,8 @@ func FeeEnabledKey(portID, channelID string) []byte {
 func KeyRelayerAddress(address string) []byte {
 	return []byte(fmt.Sprintf("%s/%s", RelayerAddressKeyPrefix, address))
 }
+
+// KeyFeeInEscrow returns the key for escrowed fees
+func KeyFeeInEscrow(channelId string, sequenceId uint64) []byte {
+	return []byte(fmt.Sprintf("%s/%s/packet/%d", FeeInEscrowPrefix, channelId, sequenceId))
+}
```

Comment:
This should include the portID in the key as well just to remain consistent with rest of IBC key structure.